README.md

```diff
@@ -80,7 +78,24 @@ information on changes between releases.
 
 ## Getting involved and contributing!
 
-Are you interested in contributing to kops? We the maintainers and community would love your suggestions, contributions, and help! We have a quickstart guide on [adding a feature](/docs/development/adding_a_feature.md) and the maintainers can be contacted at any time to learn more about how to get involved.
+Are you interested in contributing to kops? We the maintainers and community would love your suggestions, contributions, and help! We have a quick-start guide on [adding a feature](/docs/development/adding_a_feature.md) and the maintainers can be contacted at any time to learn more about how to get involved.
+
+In the interest of getting more new folks involved with kops, we are starting to tag issues with `good-starter-issue`. These are typically issues that have smaller scope but are good ways to start to get acquainted with the codebase.
+
+We also encourage ALL active community participants to act as if they are maintainers, even if you don't have the full permissions. This is still a small enough community that if you get involved and show an active interest and want to be a maintainer-- we would most likely love to have you. So show us what you can do!
```

Comment:
Not about the size of the community. No matter how big this project gets, this is the culture that we strive for. 

Can we drop the size reference, and talk about that we are here to serve, and build a community that welcomes help!